src/language-js/flow-comments.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+function hasFlowShorthandAnnotationComment(node) {
+  // https://flow.org/en/docs/types/comments/
+  // Syntax example: const r = new (window.Request /*: Class<Request> */)("");
+
+  return (
+    node.extra &&
+    node.extra.parenthesized &&
+    !!node.trailingComments &&
+    node.trailingComments[0].value.startsWith(":")
+  );
+}
+
+module.exports = hasFlowShorthandAnnotationComment;
```

Comment:
Can you rename this file to `utils.js` and export this as an object (i.e. `{ hasFlowShorthandAnnotationComment }`)?